<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BookList</class>
 <widget class="QWidget" name="BookList">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>800</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="mainLayout" stretch="0,0,1">
   <property name="spacing">
    <number>10</number>
   </property>
   <property name="leftMargin">
    <number>10</number>
   </property>
   <property name="topMargin">
    <number>10</number>
   </property>
   <property name="rightMargin">
    <number>10</number>
   </property>
   <property name="bottomMargin">
    <number>10</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="menuLayout" stretch="1,0,0">
     <property name="spacing">
      <number>10</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <spacer name="leftSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </spacer>
     </item>
     <item>
      <widget name="openFavouriteButton" class="QPushButton">
       <property name="text">
        <string>Open Favourite Book</string>
       </property>
      </widget>
     </item>
     <item>
      <widget name="refreshButton" class="QPushButton">
       <property name="text">
        <string>Refresh</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget name="divider" class="Line">
     <property name="styleSheet">
      <string notr="true">* { min-height: 1px; background: black; }</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="bookLayout">
     <property name="spacing">
      <number>10</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
    </layout>
   </item>
   <item>
    <spacer name="bottomSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets/>
 <resources/>
 <connections/>
</ui>
